### PR TITLE
restore lazy load behavior

### PR DESCRIPTION
**Pull request by bollwyvl**
Restores lazy loading (#68) removed in lab3 update (#220)

### CODE REVIEW

**Comment by ianhi** [comment]
oooh that just does it? That is much simpler than what it was before.

Thanks!

(this should one day become part of the cookiecutter probably)

**Comment by bollwyvl** [comment]
```
 118 Jan  4 22:30 style.js
4.0K Jan  4 22:30 ..
7.8K Jan  4 22:30 remoteEntry.09f4705afcb77e2f7220.js
1.3K Jan  4 22:30 981.918eb772cf59529af654.js.LICENSE.txt
 21K Jan  4 22:30 981.918eb772cf59529af654.js
450K Jan  4 22:30 878.f5b3904c8811fd52d11d.js  # <<< this right here is a bummer to block the page on
  64 Jan  4 22:30 781.feac2f1356387ca26759.js.LICENSE.txt
 23K Jan  4 22:30 781.feac2f1356387ca26759.js
 83K Jan  4 22:30 703.393ff40cb4c0f7d0b495.js
 20K Jan  4 22:30 662.31fe7bbb98cbed56c650.js
 485 Jan  4 22:30 58.5b5cc1107f4e0cc0ae9d.js.LICENSE.txt
336K Jan  4 22:30 58.5b5cc1107f4e0cc0ae9d.js
3.4K Jan  4 22:30 578.58b2d28010aa159adb8c.js
3.5K Jan  4 22:30 480.3769a7db065e12f94342.js
 13K Jan  4 22:30 367.e0d38ed2886ca12c2fcd.js
 83K Jan  4 22:30 142.af091c4e79af55e6bead.js
4.0K Jan  4 22:30 .
```

**Comment by bollwyvl** [comment]
Yeah, the webpackchunkname is purely cosmetic. Not a huge fan of the `<number>.<hash>.js` for debugging. But naming things is hard, if the name is taken!

File: src/plugin.ts [hunk 1]
```diff
@@ -38,6 +36,6 @@ function activateWidgetExtension(
   registry.registerWidget({
     name: MODULE_NAME,
     version: MODULE_VERSION,
-    exports: widgetExports,
+    exports: () => import('./widget'),
```
Comment (ianhi) [inline src/plugin.ts:39]:
```suggestion
    exports: () => import('./index'),
```

we also import from `graph` and importing from index picks up both

**Comment by ianhi** [comment]
Thanks!

**Pull request merged**